db/disk.go

```diff
@@ -44,3 +49,72 @@ func initDisks(tx *sql.Tx) error {
 
 	return err
 }
+
+func (d DB) CreateDisk(vendor string,model string,
```

Comment:
Formatting: you should always run [`go fmt`](https://blog.golang.org/go-fmt-your-code) before committing Go code.